tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -140,7 +140,7 @@ stages:
         steps:
         - task: UseRubyVersion@0
           inputs:
-            versionSpec: '= 2.4'
+            versionSpec: '= 2.5.9'
```

Comment:
Did you intentionally set 2.5.9 instead of `2.5`?

```
##[warning]It is not recommended to specify exact version on Microsoft-Hosted agents. Patch version of Ruby can be replaced by new one on Hosted agents without notice and build stops to work. it is recommended to specify only major or major and minor version (Example: `2` or `2.4`)
```

Reply:
I only picked 2.5.9 because that is what was listed in the error message; if the patch version isn't required than I'm fine with removing it.